sql/core/src/test/scala/org/apache/spark/sql/StreamTest.scala

```diff
@@ -142,7 +142,10 @@ trait StreamTest extends QueryTest with Timeouts {
   case object StopStream extends StreamAction with StreamMustBeRunning
 
   /** Starts the stream, resuming if data has already been processed.  It must not be running. */
-  case object StartStream extends StreamAction
+  case class StartStream(trigger: Trigger = null, triggerClock: Clock = null) extends StreamAction
```

Comment:
This layer of `null`s was intended to delegate the default values of `StreamExecution` into these tests, so that we don't have to set the same default values in many places and maintain their consistency. But since it seems very unlikely that we would change the default values, so I've removed the `null`s and followed your comments.

Thanks!